### PR TITLE
API departures, connector RENNES

### PR DESCRIPTION
**Pull request by julienbt**
Add a new connector for the API `departures`

### CODE REVIEW

File: internal/departures/rennes/buslines/buslines.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
+package buslines
```
Comment (kadhikari) [inline internal/departures/rennes/buslines/buslines.go:1]: Here we may have Lines of different modes like bus, metro, train. Better rename `buslines.go` to `lines.go`.
Delete preceding work Bus or bus everywhere.

File: internal/departures/rennes/scheduledtimes/scheduledtimes.go [hunk 2]
```diff
@@ -0,0 +1,91 @@
+package scheduledtimes
```
Comment (kadhikari) [inline internal/departures/rennes/scheduledtimes/scheduledtimes.go:1]: Rename as stoptimes.go and renames all concerned variables

File: internal/departures/rennes/scheduledtimes/scheduledtimes_test.go [hunk 3]
```diff
@@ -0,0 +1,75 @@
+package scheduledtimes
```
Comment (kadhikari) [inline internal/departures/rennes/scheduledtimes/scheduledtimes_test.go:1]: Same as above

File: internal/departures/rennes/routes/routes.go [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+const RoutesCsvFileName string = "chainages.pty"
+const routesCsvNumOfFields int = 5
+
+/* -----------------------------------------------------------------------------------
+// Structure and Consumer to creates Routes objects based on a line read from a CSV
+----------------------------------------------------------------------------------- */
+type Route struct {
+	Id                string // ID of the route
+	BusLineInternalId string
```
Comment (kadhikari) [inline internal/departures/rennes/routes/routes.go:20]:
```suggestion
	LineInternalId string
```

File: internal/departures/rennes/dbinternallinks/dbinternallinks_test.go [hunk 5]
```diff
@@ -0,0 +1,79 @@
+package dbinternallinks
```
Comment (kadhikari) [inline internal/departures/rennes/dbinternallinks/dbinternallinks_test.go:1]: Same as above

**Review by kadhikari — CHANGES_REQUESTED** [review]

File: internal/departures/rennes/buslines/buslines_test.go [hunk 6]
```diff
@@ -0,0 +1,71 @@
+package buslines
```
[the remark this answers is not in the document]
Comment (julienbt) [reply internal/departures/rennes/buslines/buslines_test.go:1]: Done

File: internal/departures/rennes/buslines/buslines.go (hunk 1 above)
Comment (julienbt) [reply internal/departures/rennes/buslines/buslines.go:1]: Done

File: internal/departures/rennes/routes/routes.go (hunk 4 above)
Comment (julienbt) [reply internal/departures/rennes/routes/routes.go:20]: Done

File: internal/departures/rennes/rennes.go [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+			appendedDepartures,
+		)
+	}
+	return result
+}
+
+type Departure struct {
+	DbInternalLinkId string
+	StopPointId      string
+	BusLineId        string
```
[the remark this answers is not in the document]
Comment (julienbt) [reply internal/departures/rennes/rennes.go:109]: Done

File: internal/departures/rennes/scheduledtimes/scheduledtimes.go (hunk 2 above)
Comment (julienbt) [reply internal/departures/rennes/scheduledtimes/scheduledtimes.go:1]: Done

File: internal/departures/rennes/dbinternallinks/dbinternallinks.go [hunk 8]
```diff
@@ -0,0 +1,87 @@
+package dbinternallinks
```
[the remark this answers is not in the document]
Comment (julienbt) [reply internal/departures/rennes/dbinternallinks/dbinternallinks.go:1]: Done

File: internal/departures/rennes/scheduledtimes/scheduledtimes_test.go (hunk 3 above)
Comment (julienbt) [reply internal/departures/rennes/scheduledtimes/scheduledtimes_test.go:1]: Done

File: internal/departures/rennes/dbinternallinks/dbinternallinks.go (hunk 8 above)
[the remark this answers is not in the document]
Comment (julienbt) [reply internal/departures/rennes/dbinternallinks/dbinternallinks.go:1]: @kadhikari
`routestoppoint.go` or `routestoppoints.go`?

[the remark this answers is not in the document]
Comment (julienbt) [reply internal/departures/rennes/dbinternallinks/dbinternallinks.go]: I have chosen `routestoppoints.go`

[the remark this answers is not in the document]
Comment (julienbt) [reply internal/departures/rennes/dbinternallinks/dbinternallinks.go]: Done

File: internal/departures/rennes/dbinternallinks/dbinternallinks_test.go (hunk 5 above)
Comment (julienbt) [reply internal/departures/rennes/dbinternallinks/dbinternallinks_test.go]: Done

**Review by kadhikari — APPROVED** [review]

**Pull request merged**